### PR TITLE
Templates for vendors

### PR DESCRIPTION
**Pull request by paigecalvert**
@jonquil2002, in preparation for my Replicon presentation, I created a couple more templates that vendors could use:
* Troubleshooting template (useful for vendors, though also really for anyone at Replicated writing docs)
* App configuration template (as in, a topic that would walk an enterprise user through completing the configuration screen)

Story: https://app.shortcut.com/replicated/story/59314/create-app-config-and-troubleshooting-templates-for-vendors

### CODE REVIEW

File: docs/templates/app-configuration.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+[Configuration Section 1] includes fields for configuring [a feature or service] in [App Name].
+
+To configure [Section 1]:
+
+1. For **[Field Name]**, enter [information].
+
+1. (Optional) For **[Field Name]**, enter [information].
+  <!-- Put (Optional) at the start to denote optional fields.-->
+
+### Configuration Section 2
```
Comment (jonquil2002) [inline docs/templates/app-configuration.md:49]: This heading doesn't match the one for the previous section. Maybe change this to "Configure [Section 2]".

File: docs/templates/app-configuration.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+       <td>[Field Name]</td>
+       <td>Enter [information].</td>
+     </tr>
+     <tr>
+       <td>(Optional) [Field Name]</td>
+       <td>Enter [information].</td>
+     </tr>
+   <table/>
+
+### (Optional) Configuration Section 3 <!-- Put (Optional) at the start to denote optional sections.-->
```
Comment (jonquil2002) [inline docs/templates/app-configuration.md:73]: Same comment as for the heading above.

File: docs/templates/app-configuration.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+1. For **[Field Name]**, enter [information].
+
+1. For **[Field Name]**, select one of the following options:
+   * **[Option A]**: To [do x], select **[Option A]**.
+   * **[Option B]**: To [do x], select **[Option B]**.
+   <!-- Use this bulleted list format if the user needs to
+   select between multiple options.-->
+
+<!-- ## Next Steps
```
Comment (jonquil2002) [inline docs/templates/app-configuration.md:85]: "Next Step" (singular) if there is only one next step?

File: docs/templates/troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
+# Troubleshooting
+
+<!-- This template is used for troubleshooting topics. Troubleshooting topics
+provide users with workarounds and extra details about common challenges.
+Troubleshooting topics often include specific error messages.
+
+For an example of a troubleshooting topic, see https://docs.replicated.com/enterprise/snapshots-troubleshooting-backup-restore -->
+
+# Section One <!-- Example: "Service X is Crashing"-->
```
Comment (jonquil2002) [inline docs/templates/troubleshooting.md:9]: Maybe say "Example heading:" so they don't think they need to include the word "example" in there?

File: docs/templates/troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+### Cause
+
+<!-- Background info about why the error occurs.-->
+
+### Solution
+
+<!-- Steps to resolve the issue.-->
+
+# Section One
```
Comment (jonquil2002) [inline docs/templates/troubleshooting.md:67]: This says section one but I think it's supposed to be section two. 

Also, side note, I'm finding this template a little difficult to follow with section one, issues 1 ,2, etc. even though I know how we would actually use it. But it's just hard to follow as a template - in part because it's so long. Even though I know you'll be showing them an actual example, they might find this template confusing. Do we need to guide them this much?

Comment (paigecalvert) [reply docs/templates/troubleshooting.md:67]: That's a great point, I agree. I think I'll remove the top layer of "section" headings. We can just go with issue headings, for sure. Whatcha think?

File: docs/templates/troubleshooting.md (hunk 4 above)
Comment (paigecalvert) [reply docs/templates/troubleshooting.md:9]: Ended up removing these sections all together; you're right it was overly complex

File: docs/templates/troubleshooting.md (hunk 5 above)
Comment (jonquil2002) [reply docs/templates/troubleshooting.md]: Super awesome!

**Review by jonquil2002 — APPROVED** [review]
lgtm - nice!
